samples/AgentSampleApp/AgentSampleApp.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
This is not the correct place for this.   This `samples` area is for end-to-end samples that use multiple libraries.   Please move this into the `/samples` space in your library.